src/set_options.js

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env node
```

Comment:
This is a much better way to do this going forward. Bash is nice but if you can use JS directly with json parsing it's much more future proofed.